sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Configurations that allow callers to specify details about how to execute
+    /// an Extract Summary action in a set of documents.
+    /// For example, set the model version, specify the order in which extracted
+    /// sentences are returned, and more.
+    /// </summary>
+    public class ExtractSummaryAction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExtractSummaryAction"/>
+        /// class which allows callers to specify details about how to execute
+        /// an Extract Summary action in a set of documents.
+        /// For example, set the model version, specify the order in which extracted
+        /// sentences are returned, and more.
+        /// </summary>
+        public ExtractSummaryAction()
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets a value that, if set, indicates the version of the text
+        /// analytics model that will be used to generate the result.  For supported
+        /// model versions, see operation-specific documentation, for example:
+        /// <see href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/concepts/model-versioning#available-versions"/>.
+        /// </summary>
+        public string ModelVersion { get; set; }
+
+        /// <summary>
+        /// If set, specifies the maximum limit of sentences returned in the result. Defaults to 3.
+        /// </summary>
+        public int? MaxSentenceCount { get; set; }
```

Comment:
"Max" or "Maximum"? I know we don't like abbreviations in .NET, but I'm not 100% what's the scope of this rule. For example, in Metrics Advisor, KC suggested the name `MaxPageSize` for one of our properties in listing APIs.

Reply:
Great question. 
According to the Framework Design Guidelines `..not to use abbreviations and acronyms that are not generally understood - that is, do not use anything that the large majority of people who are not experts in a given field would not know the meaning of immediately`.
Doing a quick search online, `Max` seems like one of those abbreviations that are well understood.
Also looking at our libraries looks like the majority of them are using `Max`. https://adpnetsource.azurewebsites.net/#Azure.Core/RetryOptions.cs,787df67e661c859b

Because of that, I will also go with `Max`